cezo_fl/server.py

```diff
@@ -179,8 +166,13 @@ def train_one_step(self, iteration: int) -> None:
         )
 
         if self.server_model:
+            # self.train()
```

Comment:
Why comment this out?

Reply:
wondering if this is the best place to call it

Reply:
just uncommentted, let's put self.train here first